src/components/info/templates/default-template.vue

```diff
@@ -0,0 +1,152 @@
+<script setup lang="ts">
+import { defineProps } from 'vue'
+import {
+  FeatureInfoJSON,
+  FeatureJSON,
+} from '../../../services/info/feature-info.model'
+import { useTranslation } from 'i18next-vue'
+import {
+  hasAttributes,
+  isEmpty,
+  isLink,
+  showAttributesByLang,
+  hasValidFID,
+  prefixKeys,
+  getTrustedUrl,
+} from './template-utilities'
+import i18next from 'i18next'
+import ProfileFeatureInfo from '@/components/info/profile-feature-info.vue'
+
+defineProps({
+  layers: {
+    type: Object as () => FeatureInfoJSON,
+    required: true,
+  },
+})
+defineEmits<{
+  (e: 'export', payload: { feature: FeatureJSON; format: 'kml' | 'gpx' }): void
+}>()
+const { t } = useTranslation()
+const currentUrl = window.location.href
+</script>
+<template>
+  <div class="flex flex-col">
+    <div>
+      <h1 class="lux-poi-title" data-cy="defaultTemplateTitle">
+        {{ t(`${layers.layerLabel}`) }}
+      </h1>
+      <div
+        v-for="feature in layers.features"
+        :key="feature.id"
+        class="lux-poi-feature"
+      >
+        <h4 v-if="feature.attributes.label">
+          {{ t(`${feature.attributes.label}`) }}
+        </h4>
+        <div data-cy="defaultTemplateAttributes" v-if="hasAttributes(feature)">
+          <div
+            v-for="entry in prefixKeys(feature.attributes, 'f_').sort(
+              (a, b) => {
+                return layers.ordered ? 0 : a.key.localeCompare(b.key)
+              }
+            )"
+            :key="entry.key"
+          >
+            <span
+              v-if="
+                !isEmpty(entry['value']) &&
+                showAttributesByLang(entry, layers.layer, i18next.language)
+              "
+            >
+              <label v-if="!isLink(entry['value'])"
+                >{{ t(`${entry['key']}`) }} :
+              </label>
+              <span
+                v-if="!isLink(entry['value'])"
+                v-dompurify-html="entry['value']"
+              ></span>
+              <a
+                v-if="
+                  !(
+                    layers.layerLabel.startsWith(
+                      'myenergy_solarkataster_luxemburg'
+                    ) || layers.layerLabel.startsWith('eau_new_Wasserstand')
+                  ) && isLink(entry['value'])
+                "
+                :href="`${entry['value']}`"
+                target="_blank"
+                >{{ t(`${entry['key']}`) }}</a
+              >
+              <a
+                data-cy="defaultTemplateSolarLink"
+                v-if="
+                  layers.layerLabel.startsWith(
+                    'myenergy_solarkataster_luxemburg'
+                  ) &&
+                  isLink(entry['value']) &&
+                  entry['key'] == 'f_href'
```

Comment:
`entry['key'] == 'f_href'`?
I don't understand these v-if conditions haha ... maybe if this was in a well named function it would be much easier to understand.
But not very important, if you don't have time for that keep as is.

Reply:
You're right, I've refactored the entire template to make it more readable.